Database/DBconnect.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose")
+
+const mySecret = process.env['MongoDBCredentials']
+
+const uri = mySecret;
+
+const initializeDBconnection = async () => {
+  try{
+    const response = await mongoose.connect(uri, {
+      useNewUrlParser: true, 
+      useUnifiedTopology: true
+    })
+    if(response){
+      console.log("MONGOOSE Connected successfuly")
+    }
```

Comment:
mongoose.connect will always return a response. If you want to check if mongoose has connected, you can try checking the `mongoose.connection.readyState` property. https://masteringjs.io/tutorials/mongoose/connection-status

Reply:
Thanks. I will look into it.